PEPit/examples/continuous_time_models/gradient_flow_convex.py

```diff
@@ -0,0 +1,122 @@
+from PEPit import PEP
+from PEPit.functions.convex_function import ConvexFunction
+
+
+def wc_gradient_flow_convex(t, verbose=1):
+    """
+    Consider the convex minimization problem
+
+    .. math:: f_\\star \\triangleq \\min_x f(x),
+
+    where :math:`f` is convex.
+
+    This code computes a worst-case guarantee for a **gradient** flow.
+    That is, it verifies the following inequality
+
+    .. math:: \\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0,
+
+    is valid, where :math:`\\mathcal{V}(X_t, t) = t(f(X_t) - f(x_\\star)) + \\frac{1}{2} \\|X_t - x_\\star\\|^2`,
+    :math:`X_t` is the output of the **gradient** flow, and where :math:`x_\\star` is the minimizer of :math:`f`.
+    In short, for given values of :math:`t`, it verifies :math:`\\frac{d}{dt}\\mathcal{V}(X_t, t)\\leqslant 0`.
+
+    **Algorithm**:
+    For :math:`t \\geqslant 0`,
+
+                .. math:: \\frac{d}{dt}X_t = -\\nabla f(X_t),
+
+    with :math:`X_{0}:= x_0 \\in \\mathbf{R}^d`.
+
+    **Theoretical guarantee**:
+
+        The following **tight** guarantee can be found in [1, p. 7]:
+
+        .. math:: \\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0.
+
+        After integrating between :math:`0` and :math:`t`,
```

Comment:
integrate between 0 and T (capital) ? to differentiate between t and T?
minor, though!